test/mp_correct/inc/taskloop_ice.mk

```diff
@@ -0,0 +1,29 @@
+#
+# Copyright (c) 2019, Arm Ltd.  All rights reserved.
```

Comment:
Do you mind removing them ? This will make tests less noisy and uniform.
Perhaps, if time permits and everyone agrees we should remove these from every test file?
@gklimowicz any comments on this ?

Reply:
Sure, I will remove.